clang/lib/Basic/Targets.cpp

```diff
@@ -247,12 +247,8 @@ std::unique_ptr<TargetInfo> AllocateTarget(const llvm::Triple &Triple,
     switch (os) {
     case llvm::Triple::Linux:
       return std::make_unique<LinuxTargetInfo<ARMbeTargetInfo>>(Triple, Opts);
-    case llvm::Triple::FreeBSD:
```

Comment:
OK - there are no supported FreeBSD releases left that support armeb, and it was uncommon enough that I can't imagine anyone wants to use contemporary clang to target unsupported releases.

Reply:
Ya, even with support being removed I didn't think it was common at all.